### ISSUE TITLE
DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`.

### ISSUE THREAD

**Issue by miker985** [Roche/pyreadstat#160]
numpy 1.20 deprecated `numpy.float` and many other aliases to the built-in types.

This emits a DeprecationWarning on import of pyreadstat

To Reproduce

```bash
cat <<EOF > test_import_DeprecationWarning.py
import pyreadstat
EOF

pytest -W error test_import_DeprecationWarning.py
```

Expected behavior
Update code to use `float` instead of `np.float` or `numpy.float`

Setup Information:
How did you install pyreadstat? pip
Platform: linux 64 bit
Python Version: 3.7.10
Python Distribution (: plain
Using Virtualenv or condaenv? virtualenv

**Comment by ofajardo** [comment]
thanks for the reproducible issue. It is solved on dev and will be there on the next release.

**Comment by ofajardo** [comment]
solved in version 1.1.5
